packages/core-components/src/lib/unsorted/viz/Column.svelte

```diff
@@ -66,7 +66,8 @@
 	export let deltaSymbol = true;
 	deltaSymbol = deltaSymbol === 'true' || deltaSymbol === true;
 
-	let options = {
+	let options;
```

Comment:
Is this necessary now you have the `$: options `

Reply:
TIL: nope!